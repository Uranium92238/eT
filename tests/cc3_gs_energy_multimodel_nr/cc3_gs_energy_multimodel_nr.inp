system
   name: H2O He
   charge: 0
end system

method
  hf
  cc3
end method

print
   output print level: verbose
end print

memory
   available: 8
end memory

solver cholesky
   threshold: 1.0d-12
end solver cholesky

solver scf
   algorithm:          scf-diis
   energy threshold:   1.0d-11
   gradient threshold: 1.0d-11
end solver scf

solver cc gs
   energy threshold: 1.0d-11
   omega threshold:  1.0d-11
end solver cc gs

do
   ground state
end do

geometry
basis: cc-pVDZ
 H          0.86681        0.60144        5.00000
 H         -0.86681        0.60144        5.00000
 O          0.00000       -0.07579        5.00000
 He         0.10000       -0.02000        7.53000
end geometry

