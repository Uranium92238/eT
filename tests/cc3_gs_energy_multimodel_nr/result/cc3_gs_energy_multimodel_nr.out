


                     eT 1.9 - an electronic structure program

  ------------------------------------------------------------------------
   Author list in alphabetical order:
  ------------------------------------------------------------------------
   J. H. Andersen, A. Balbi, S. Coriani, S. D. Folkestad, T. Giovannini, 
   L. Goletto, T. S. Haugland, A. Hutcheson, I-M. Høyvik, E. F. Kjønstad, 
   H. Koch, R. Matveeva, T. Moitra, R. H. Myhre, A. C. Paul, S. Roet, 
   E. Ronca, M. Scavino, A. K. Schnack-Petersen, A. S. Skeidsvoll, Å. 
   H. Tveten
  ------------------------------------------------------------------------
   J. Chem. Phys. 152, 184103 (2020); https://doi.org/10.1063/5.0004713


   This is eT 1.9.0 Jazzy
  ------------------------------------------------------------
  Configuration date: 2022-10-19 10:25:53 UTC -07:00
  Git branch:         detached HEAD
  Git hash:           57c35dcc31df636729251e2448b7b0d0dd22e1f1
  Fortran compiler:   GNU 11.3.0
  C compiler:         GNU 11.3.0
  C++ compiler:       GNU 11.3.0
  LAPACK type:        SYSTEM_NATIVE
  BLAS type:          SYSTEM_NATIVE
  64-bit integers:    OFF
  OpenMP:             ON
  PCM:                OFF
  Forced batching:    OFF
  Runtime checks:     OFF
  ------------------------------------------------------------


  :: Input file
  ================

     Note: geometry section is excluded from this print

     system
        name: h2o he
        charge: 0
     end system

     method
       hf
       cc3
     end method

     print
        output print level: verbose
     end print

     memory
        available: 8
     end memory

     solver cholesky
        threshold: 1.0d-12
     end solver cholesky

     solver scf
        algorithm:          scf-diis
        energy threshold:   1.0d-11
        gradient threshold: 1.0d-11
     end solver scf

     solver cc gs
        energy threshold: 1.0d-11
        omega threshold:  1.0d-11
     end solver cc gs

     do
        ground state
     end do


  Calculation start:2022-10-19 10:30:12 UTC -07:00

  Running on 2 OMP threads
  Memory available for calculation: 8.000000 GB


  :: RHF wavefunction
  ===================

  Will perform Coulomb and exchange terms in the Fock matrix collectively

  Libint electron repulsion integral precision:  0.1000E-23

     ==============================================================================
                                     Geometry (angstrom)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1  H     0.866810000000     0.601440000000     5.000000000000        1
        2  H    -0.866810000000     0.601440000000     5.000000000000        2
        3  O     0.000000000000    -0.075790000000     5.000000000000        3
        4 He     0.100000000000    -0.020000000000     7.530000000000        4
     ==============================================================================

     ==============================================================================
                                     Geometry (a.u.)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1  H     1.638033502034     1.136556880358     9.448630622825        1
        2  H    -1.638033502034     1.136556880358     9.448630622825        2
        3  O     0.000000000000    -0.143222342981     9.448630622825        3
        4 He     0.188972612457    -0.037794522491    14.229637717975        4
     ==============================================================================

  - Cholesky decomposition of AO overlap to get linearly independent AOs:

     Linear dependence threshold:             0.10E-05
     Number of atomic orbitals:               29
     Number of orthonormal atomic orbitals:   29

  - Molecular orbital details:

     Number of occupied orbitals:         6
     Number of virtual orbitals:         23
     Number of molecular orbitals:       29


  Generating initial SAD density
  ==============================
     Generated atomic density for H  using UHF/cc-pvdz
     Generated atomic density for O  using UHF/cc-pvdz
     Generated atomic density for He using UHF/cc-pvdz


  Determining reference state
  ===========================

  - Setting initial AO density to sad

     Energy of initial guess:               -78.492022836321
     Number of electrons in guess:           12.000000000000

  - Screening and integral thresholds:

     Coulomb screening threshold:    0.1000E-16
     Exchange screening threshold:   0.1000E-14
     ERI cutoff:                     0.1000E-16
     One-electron integral  cutoff:  0.1000E-21
     Cumulative Fock threshold:      0.1000E+01

  - SCF solver settings:

     Maximum iterations:                   100
     Acceleration type:                   diis

  - Convergence thresholds

     Residual threshold:            0.1000E-10
     Energy threshold:              0.1000E-10

  - DIIS tool settings:

     DIIS dimension:   8

     Storage (solver scf_errors): memory
     Storage (solver scf_parameters): memory

  Iteration       Energy (a.u.)      Max(grad.)    Delta E (a.u.)
  ---------------------------------------------------------------
     1           -78.796606592585     0.9786E-01     0.7880E+02
  Fock matrix construction using density differences
     2           -78.828675852654     0.7077E-01     0.3207E-01
  Fock matrix construction using density differences
     3           -78.843487343819     0.6747E-02     0.1481E-01
  Fock matrix construction using density differences
     4           -78.843814479549     0.2753E-02     0.3271E-03
  Fock matrix construction using density differences
     5           -78.843850612079     0.3973E-03     0.3613E-04
  Fock matrix construction using density differences
     6           -78.843851670925     0.5220E-04     0.1059E-05
  Fock matrix construction using density differences
     7           -78.843851692779     0.6096E-05     0.2185E-07
  Fock matrix construction using density differences
     8           -78.843851693528     0.2137E-05     0.7487E-09
  Fock matrix construction using density differences
     9           -78.843851693630     0.3151E-06     0.1027E-09
  Fock matrix construction using density differences
    10           -78.843851693631     0.2637E-07     0.4974E-12
  Fock matrix construction using density differences
    11           -78.843851693631     0.5634E-08     0.1421E-13
  Fock matrix construction using density differences
    12           -78.843851693631     0.1415E-08     0.1421E-13
  Fock matrix construction using density differences
    13           -78.843851693631     0.6806E-09     0.4263E-13
  Fock matrix construction using density differences
    14           -78.843851693631     0.1491E-09     0.2842E-13
  Fock matrix construction using density differences
    15           -78.843851693631     0.2501E-10     0.7105E-13
  Fock matrix construction using density differences
    16           -78.843851693631     0.5168E-11     0.4263E-13
  ---------------------------------------------------------------
  Convergence criterion met in 16 iterations!

  - Summary of RHF wavefunction energetics (a.u.):

     HOMO-LUMO gap:                  0.645629080245
     Nuclear repulsion energy:      12.116100574587
     Electronic energy:            -90.959952268218
     Total energy:                 -78.843851693631

  No frozen fock contributions!

  Libint electron repulsion integral precision:  0.1000E-23


  :: CC3 wavefunction
  ===================

     Bath orbital(s):         False

   - Number of orbitals:

     Occupied orbitals:    6
     Virtual orbitals:     23
     Molecular orbitals:   29
     Atomic orbitals:      29

   - Number of ground state amplitudes:

     Single excitation amplitudes:  138
     Double excitation amplitudes:  9591


  Cholesky-decomposing electron repulsion integrals
  =================================================

  Cholesky decomposition of electronic repulsion integrals solver
  ---------------------------------------------------------------

  Performs a Cholesky decomposition of the two-electron electronic repulsion 
  integrals in the atomic orbital basis,

  (ab|cd) = sum_J L_ab^J L_cd^J.

  Once the Cholesky basis has been determined, the vectors L^J are constructed 
  and stored to disk. These may either be used directly, or be transformed 
  to the MO basis for use in post-HF calculations. For more information, 
  see S. D. Folkestad, E. F. Kjønstad and H. Koch, JCP, 150(19), (2019)

  - Cholesky decomposition settings:

     Target threshold is:   0.10E-11
     Span factor:           0.10E-01
     Max qual:                  1000

  - Cholesky decomposition ao details:

     Total number of AOs:                    29
     Total number of shell pairs:           120
     Total number of AO pairs:              435

  - Preparing diagonal for decomposition:

     Significant shell pairs:               118
     Significant AO pairs:                  431

     Construct shell pairs:                 120
     Construct AO pairs:                    435

  Iter.  #Sign. ao pairs / shell pairs   Max diagonal    #Qualified    #Cholesky    Cholesky array size
  -------------------------------------------------------------------------------------------------------
     1               406 /     108       0.47383E+01         147             42             17052
     2               319 /      93       0.47165E-01         234            111             35409
     3               250 /      76       0.46944E-03         178            183             45750
     4               187 /      55       0.38270E-05         145            265             49555
     5                87 /      25       0.38106E-07          78            324             28188
     6                21 /       3       0.37202E-09          43            349              7329
     7                 0 /       0       0.36652E-11           7            354                 0
  -------------------------------------------------------------------------------------------------------

  - Summary of Cholesky decomposition of electronic repulsion integrals:

     Final number of Cholesky vectors: 354

 - Testing the Cholesky decomposition decomposition electronic repulsion integrals:

     Maximal difference between approximate and actual diagonal:              0.7161E-12
     Minimal element of difference between approximate and actual diagonal:  -0.1238E-14

  - Settings for integral handling:

     Cholesky vectors in memory: True
     ERI matrix in memory:       True

  - Finished decomposing the ERIs.

     Total wall time (sec):              0.35200
     Total cpu time (sec):               0.24163


  Determining CC cluster amplitudes
  =================================

   - Davidson tool settings:

     Number of parameters:                 9729
     Number of requested solutions:           1
     Max reduced space dimension:            50

     Storage (newton_raphson_amplitude_updator_trials): file
     Storage (newton_raphson_amplitude_updator_transforms): file

   - DIIS coupled cluster ground state solver
  ----------------------------------------------

  A DIIS CC ground state amplitude equations solver. It uses an extrapolation 
  of previous quasi-Newton perturbation theory estimates of the next amplitudes. 
  See Helgaker et al., Molecular Electronic Structure Theory, Chapter 
  13.

  - CC ground state solver settings:

  - Convergence thresholds

     Residual threshold:            0.1000E-10
     Energy threshold:              0.1000E-10
     Max number of iterations:      100

  - DIIS tool settings:

     DIIS dimension:   8

     Storage (cc_gs_diis_errors): file
     Storage (cc_gs_diis_parameters): file

  Iteration    Energy (a.u.)        |omega|       Delta E (a.u.)
  ---------------------------------------------------------------
    1           -79.084241931608     0.9358E-01     0.7908E+02

     Iteration     Residual norm
     -----------------------------
       1           0.2284E-01
       2           0.5837E-02
       3           0.1639E-02
       4           0.5953E-03
     -----------------------------

    2           -79.103956188886     0.1780E-02     0.1971E-01

     Iteration     Residual norm
     -----------------------------
       1           0.5257E-03
       2           0.1805E-03
       3           0.4898E-04
       4           0.1928E-04
       5           0.5749E-05
     -----------------------------

    3           -79.104372326749     0.1172E-03     0.4161E-03

     Iteration     Residual norm
     -----------------------------
       1           0.3683E-04
       2           0.1404E-04
       3           0.3164E-05
       4           0.1200E-05
       5           0.4234E-06
     -----------------------------

    4           -79.104388195848     0.9192E-05     0.1587E-04

     Iteration     Residual norm
     -----------------------------
       1           0.3235E-05
       2           0.1119E-05
       3           0.2315E-06
       4           0.5651E-07
     -----------------------------

    5           -79.104388536633     0.8539E-06     0.3408E-06

     Iteration     Residual norm
     -----------------------------
       1           0.2879E-06
       2           0.8616E-07
       3           0.1732E-07
       4           0.4841E-08
     -----------------------------

    6           -79.104388533341     0.4169E-07     0.3292E-08

     Iteration     Residual norm
     -----------------------------
       1           0.1104E-07
       2           0.4435E-08
       3           0.9915E-09
       4           0.3044E-09
     -----------------------------

    7           -79.104388533080     0.2159E-08     0.2611E-09

     Iteration     Residual norm
     -----------------------------
       1           0.4881E-09
       2           0.1666E-09
       3           0.4347E-10
       4           0.1487E-10
     -----------------------------

    8           -79.104388533034     0.9443E-10     0.4638E-10

     Iteration     Residual norm
     -----------------------------
       1           0.1961E-10
       2           0.5795E-11
       3           0.1717E-11
       4           0.7620E-12
     -----------------------------

    9           -79.104388533033     0.3531E-11     0.9379E-12
  ---------------------------------------------------------------
  Convergence criterion met in 9 iterations!

  - Ground state summary:

     Final ground state energy (a.u.):   -79.104388533033

     Correlation energy (a.u.):           -0.260536839402

     Largest single amplitudes:
     -----------------------------------
        a       i         t(a,i)
     -----------------------------------
        1      5        0.015274304435
       14      4       -0.008970023658
        7      4        0.007312583244
        4      5        0.006872684673
        2      4        0.006012921703
       15      5       -0.005704150040
        6      2        0.005025576014
       13      5        0.004933889031
        5      6        0.004366074635
        3      4        0.004126391914
     ------------------------------------

     Largest double amplitudes:
     --------------------------------------------------
        a      i       b      j         t(ai,bj)
     --------------------------------------------------
        2      4       2      4       -0.049931914505
        5      6       5      6       -0.046567925291
        9      3       9      3       -0.041349512985
        3      4       3      4       -0.037778342513
        1      5       1      5       -0.037299297231
        6      5       6      5       -0.034903296771
       16      3      16      3       -0.032107572471
       17      3      17      3       -0.032051843284
        2      4       1      5       -0.031565942819
       18      3      18      3       -0.031349888894
     --------------------------------------------------

  - Finished solving the CC3 ground state equations

     Total wall time (sec):              2.11200
     Total cpu time (sec):               1.19483

  - Cleaning up cc3 wavefunction

  Peak memory usage during the execution of eT: 14.673696 MB

  Total wall time in eT (sec):              3.05500
  Total cpu time in eT (sec):               2.01691

  Calculation end:2022-10-19 10:30:16 UTC -07:00

  - Implementation references:

     eT: https://doi.org/10.1063/5.0004713
     CC3: https://doi.org/10.1021/acs.jctc.0c00686
     Cholesky decomposition of ERIs: https://doi.org/10.1063/1.5083802
     Multimodel Newton algorithm: https://doi.org/10.1063/5.0010989

  eT terminated successfully!
